content/en/resources/conversations-with-my-friends-in-ai-podcast.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Conversations With My Friends in AI Podcast"
+slug: "conversations-with-my-friends-in-ai-podcast-appearance"
+draft: false
+event_date: "2024-01-23"
+image: "img/resources/resource-image-for-podcasts.webp"
```

Comment:
I noticed that @pdeziel used DALL-E to generate images for the LLM blog posts. I'm thinking of using that to replace this image in the future.